src/Playlist.js

```diff
@@ -645,7 +686,7 @@ export default class {
         let activeTrack = this.getActiveTrack();
         let trackElements = this.tracks.map((track) => {
             return track.render(this.getTrackRenderData({
-                "isActive": (activeTrack === track) ? true : false,
+                "isActive": (this.getTimeSelection().start!=this.getTimeSelection().end)?((activeTrack === track) ? true : false):true,
```

Comment:
It might be good to make a boolean function for line vs chunk selection since this is used twice now.